addon/components/hyper-table/cell.js

```diff
@@ -111,6 +116,13 @@ export default Component.extend({
     toggleFiltersPanel() {
       this.toggleProperty('showFiltersPanel');
       this.set('manager.applyingFiltersOn', this.column.property);
+    },
+    filterColumn() {
+      this.manager.updateOrdering(this.column, this.column.orderBy ? 
+        this.column.orderBy == orderingOptions.asc ? 
+        null
+        : orderingOptions.asc 
+        : orderingOptions.desc);
```

Comment:
i think it will be more readable if you split those conditions. ternary conditions look weird when chained like that 😅 

maybe something like:
```
let nextDirection = this.column.orderDirection === 'asc' ? 'desc' : 'asc';
this.manager.updateOrdering(this.column, `${this.column.orderType}:${nextDirection`);
```

(you will have to implement `orderType` on the Column in `addon/types/column.js`, it will look a lot like orderDirection)